CMSgov/design-system
# [WNMGDS-2343] Migrate HCgov logo details from styleguide.healthcare.gov

## Summary

- Migrate content from this page to the doc site https://styleguide.healthcare.gov/design/identity 

## How to test

1. Go to https://cmsgov.github.io/design-system/branch/WNMGDS-2343-Hcgov-logo/components/logos/healthcare-logo/?theme=core and see that the content has been migrated from https://styleguide.healthcare.gov/design/identity 
2. Note that the "x" next to the incorrect image examples will be red in production (found a bug about that today)

## Checklist




### If this is a change to code:


### If this is a change to documentation:
